app/package.json

```diff
@@ -31,7 +31,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "eject": "react-scripts eject",
-    "gql-gen": "graphql-codegen --config codegen.js --require dotenv/config"
+    "gql-gen": "graphql-codegen --config codegen.js --require dotenv/config",
+    "lint": "eslint --ext ts,tsx src && tsc --noEmit && prettier --check \"src/{**/*,*}.{js,ts,jsx,tsx}\""
```

Comment:
I was struggling to determine how to exclude `src/generated` from this glob pattern which is the reason `prettier` is still failing.